package.json

```diff
@@ -16,9 +16,10 @@
     "format": "prettier --write",
     "lint-ts": "tsc --noEmit --skipLibCheck && eslint --ext .json,.js,.ts,.tsx .",
     "lint-css": "stylelint \"src/**/*.css\"",
-    "link-check": "./scripts/link-check-git-all.sh",
-    "link-check-diff": "./scripts/link-check-git-diff.sh",
-    "link-check-exclude": "./scripts/exclude-links-check.sh"
+    "link-check": "repo-link-check -c config/link-check/config.yml -s filesystem -u",
```

Comment:
what does `-u` mean?

Reply:
`-u` is shorthand for `--report-unused-patterns`, [Here's a link to the README section that lists the CLI flags with shorthands](https://github.com/iterative/link-check#link-check-option-flags)

Reply:
let's use explicit long name?

also, `-u only` is very confusing- I don't quite understand what does it do from the docs

Reply:
`report-unused-patterns` is now reworked into `unused-patterns-only`.
Now unused link patterns are always reported, and `-u / --unused-patterns-only` makes it so the program exclusively checks for unused patterns using a dry run.

I also used longhand `--unused-patterns-only` in the configuration here.